docs/pages/docs/custom-schemas/custom-blocks.mdx

```diff
@@ -77,76 +41,108 @@ const Alert = createReactBlockSpec(
 );
 ```
 
-You can see that `createReactBlockSpec` takes two object arguments:
+#### Block Config (`CustomBlockConfig`)
 
-#### `blockConfig`
+The Block Config describes the shape of the Block, it's type and the properties it supports:
```

Comment:
Think we should just explicitly say the Block Config defines the block's name, properties, and type of content it contains